spec/unit/source/git_spec.rb

```diff
@@ -122,7 +130,11 @@
   describe "generate_environments respects filter_command setting and name" do
     before do
       allow(subject.cache).to receive(:branches).and_return ['master', 'development', 'production']
-      subject.instance_variable_set(:@filter_command, '[ $R10K_NAME = mysource ]')
+      if R10K::Util::Platform.windows?
+        let(:filter_command) { 'powershell.exe if ($env:R10K_BRANCH.equals(\"mysource\")) {exit 1} else {exit 0}' }
```

Comment:
Should these exit codes be reversed for this case? since the non-windows version wants it to be `mysource`.